vidyut-lipi/src/bin/transliterate.rs

```diff
@@ -0,0 +1,35 @@
+use clap::Parser;
+use std::error::Error;
+use vidyut_lipi::transliterate;
+
+#[derive(Parser)]
+#[command(author, version, about)]
+struct Args {
+    /// Text to transliterate
+    #[arg(long)]
+    text: String,
+}
+
+fn run(args: Args) -> Result<(), Box<dyn Error>> {
+    let result = transliterate(
+        &args.text,
+        vidyut_lipi::Scheme::Iast,
+        vidyut_lipi::Scheme::Slp1,
+    );
+
+    println!("Transliterated text is {}", result);
```

Comment:
I think it's cleaner to print just the output, so that the `Transliterated text is ` isn't confused for part of the output